update_version.sh

```diff
@@ -7,9 +7,7 @@ dch_installed=$?
 command -v git > /dev/null
 git_installed=$?
 if [ $dch_installed -ne 0 ] || [ $git_installed -ne 0 ]; then
-  echo "You must run this on a system with dch and git available (in order to edit the Debian package changelog and commit message)"
-  echo "If you are on Debian/Ubuntu, apt-get install devscripts git"
-  exit 1
+  sudo apt-get install -y devscripts git
```

Comment:
I admit this is *incredibly* half-assed, but part of the reasoning for checking this stuff was to make sure that this script is only run in the Vagrant build machine (the comment kinda indicates that that was the goal).

Would you mind adding a test before this change to that effect? Something like "if $(whoami) is not "vagrant", exit with an explanatory message about only running this in the build VM".